lib/config.yml

```diff
@@ -124,6 +124,9 @@ Lint/RedundantSafeNavigation:
 Lint/StructNewOverride:
   Enabled: true
 
+Lint/DuplicateBranch:
```

Comment:
I have never had this catch a legitimate error, but it frequently catches me when I want to separate clauses for clarity.